### PR TITLE
[16.0][MIG] purchase_order_supplierinfo_update: Migration to version 16.0

### PR DESCRIPTION
**Pull request by Yadier-Tecnativa**
@Tecnativa TT42424

### CODE REVIEW

**Comment by Yadier-Tecnativa** [comment]
ping @sergio-teruel  @pedrobaeza

File: README.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ addon | version | maintainers | summary
 [purchase_order_approved](purchase_order_approved/) | 16.0.1.0.0 |  | Add a new state 'Approved' in purchase orders.
 [purchase_order_line_menu](purchase_order_line_menu/) | 16.0.1.1.0 |  | Adds Purchase Order Lines Menu
 [purchase_order_owner](purchase_order_owner/) | 16.0.1.0.0 |  | Purchase Order Owner
+[purchase_order_product_recommendation](purchase_order_product_recommendation/) | 16.0.1.0.0 |  | Recommend products to buy to supplier based on history
```
Comment (pedrobaeza) [inline README.md]: These files from purchase_order_product_recommendation shouldn't be touched.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
The rest is correct

**Review by PavarunCh — APPROVED** [review]
LGMT :+1:

**Comment by pedrobaeza** [comment]
@Yadier-Tecnativa a new merge commit has appeared, which is incorrect.

**Review by sergio-teruel — APPROVED** [review]
Functional review

**Comment by sergio-teruel** [comment]
ping @juangomezzurita

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**